ui/src/app/app-routing.module.ts

```diff
@@ -22,6 +24,14 @@ const routes: Routes = [
     path: '',
     component: HomePageComponent,
     data: {title: 'View Workspaces'}
+  }, {
+    path: 'admin-review-workspace',
```

Comment:
Maybe we can start an `admin` URL stub - so these paths would be `admin/review-workspace` and `admin/id-verification` and future "administrative" type things can go under the `admin` root.

Reply:
I added the admin-review-... to the path (and the components on advice from @blrubenstein. I'm open to either method.

Reply:
I like John's suggestion here.

Reply:
Agreed. And we should probably move workspace-review to be under the admin as well. (Maybe in a cleanup?)